From 56d37ac1b0bd7e3230096341aa894effa062fd62 Mon Sep 17 00:00:00 2001
From: NIIBE Yutaka <gniibe@fsij.org>
Date: Wed, 22 Aug 2018 13:49:13 +0900
Subject: [PATCH 4/4] target: cortex-m: More wait tick.

--
This is needed for sleeping FST-01G.  I don't know if 80 is
relevant, but it works at least.
---
 src/target/cortex_m.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/target/cortex_m.c b/src/target/cortex_m.c
index 6a0ffdad..4201e984 100644
--- a/src/target/cortex_m.c
+++ b/src/target/cortex_m.c
@@ -1927,7 +1927,7 @@ int cortex_m_examine(struct target *target)
 		}
 
 		/* Leave (only) generic DAP stuff for debugport_init(); */
-		armv7m->debug_ap->memaccess_tck = 8;
+		armv7m->debug_ap->memaccess_tck = 80;
 
 		retval = mem_ap_init(armv7m->debug_ap);
 		if (retval != ERROR_OK)
-- 
2.18.0

